### ISSUE TITLE
feat: Push 4.3.3 on DockerHub

### ISSUE THREAD

**Issue by cesar-lizurey** [jekyll/jekyll#9678]
## Summary

v4.3.3 is not on Dockerhub: https://hub.docker.com/r/jekyll/jekyll/tags

## Motivation

I'd like to automate my CI/CD with the latest Jekyll version.

## Guide-level explanation

For security reasons, I think a lot of people would like to upgrade to 4.3.3

## Reference-level explanation

Do the same thing as for the 4.2.2 which is on DockerHub

## Drawbacks

You should not do this if you want Jekyll to disappear!

## Unresolved Questions

Why are we on Earth?

**Comment by ashmaroli** [comment]
Hello @cesar-lizurey,
The docker images are exclusively managed by `envygeeks` who is no longer an active Jekyll maintainer.

So, I'm afraid this won't be resolved until envygeeks decides so.

**Comment by stefandecimelli** [comment]
> Hello @cesar-lizurey, The docker images are exclusively managed by `envygeeks` who is no longer an active Jekyll maintainer.
> 
> So, I'm afraid this won't be resolved until envygeeks decides so.

What is the plan going forward for docker images in this case? I was about to open a similar issue needing a docker image for 3.10 since this is the currently supported version for Github Pages: https://pages.github.com/versions/

**Comment by F21** [comment]
Should jekyll take over the jekyll org on Docker hub? The org on Docker hub states that it produces the official docker images for jekyll, but the owner no longer maintains the images and latest tagged version is quite outdated.

**Comment by mattr-** [comment]
I'm not sure that we **can** take it over. I'll look into it.

**Comment by F21** [comment]
Perhaps another option would be to have jekyll produce docker images and publish them to ghcr.

**Comment by nathanweeks** [comment]
Another option would be to submit a proposal for (and subsequently maintain) a Jekyll [Docker Official Image](https://docs.docker.com/docker-hub/repos/manage/trusted-content/official-images/). The image would be downloaded (e.g., using the Docker CLI) with `docker pull jekyll` (short name) or`docker pull docker.io/library/jekyll` (fully-qualified name), instead of `docker pull jekyll/jekyll` / `docker pull docker.io/jekyll/jekyll`

**Comment by chimbori** [comment]
Curious if there has been any movement here?

As the official project, it should be fairly straightforward to request control of the official channel from Docker Hub admins if the previous sole maintainer has abandoned the project.

Since this particular Docker image is blessed in Jekyll docs as the canonical official one, it’s important to keep it updated with the latest, and in full control of currently-active maintainers.

Thanks!

**Comment by oDinZu** [comment]
also curious about this @envygeeks

**Comment by simonvanderveldt** [comment]
> Should jekyll take over the jekyll org on Docker hub? The org on Docker hub states that it produces the official docker images for jekyll, but the owner no longer maintains the images and latest tagged version is quite outdated.

Definitely! Having the official Docker image not be maintained by the team itself is pretty suboptimal.

@mattr- commented here https://github.com/envygeeks/jekyll-docker/issues/364#issuecomment-2711137870 that the Jekyll org/team already has access to the Jekyll Docker hub organization, so this should be relatively easy.

@mattr- Any updates on this? Are there any challenges with making this happen?
